memo.rb

```diff
@@ -1,17 +1,28 @@
 # frozen_string_literal: true
 
 require 'sinatra'
-require 'json'
+require 'pg'
 
-
-MEMOS_FILE = './memos.json'
+CONN = PG.connect(dbname: 'sinatra_memo_db')
 
 def load_memos
-  File.open(MEMOS_FILE) { |file| JSON.parse(file.read) }
+  CONN.exec('SELECT * FROM memos ORDER BY id')
+end
+
+def load_memo(id)
+  CONN.exec_params('SELECT * FROM memos WHERE id = $1', [id]).first
+end
+
+def make_memo(title, content)
+  CONN.exec_params('INSERT INTO memos (title, content) VALUES ($1, $2)', [title, content])
```

Comment:
```suggestion
def make_memo(params)
  CONN.exec_params('INSERT INTO memos (title, content) VALUES ($1, $2)', params.values_at(:title, :content))
```

項目が多いとどんどん引数が増えていってしまうのでハッシュを受け取る方が効率的かも